lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeRecordCursor.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Mention in the commit message what issue does it solve. And please explain the problem.